### ISSUE TITLE
fromJSON() and toJSON() with Architect.Perceptron

### ISSUE THREAD

**Issue by vishiswoz** [cazala/synaptic#189]
Current I'm doing this
`var network = new Architect.Perceptron(inputs, 5, 5, outputs);`

Then I do 
`var networkJSON = network.toJSON()`

Except now I can't do 
`var originalNetwork = Network.fromJSON(networkJSON)`

Because `originalNetwork` is a `Network` and I want a `Architect`, or will there be no differences between the 2? Sorry if this seems like a stupid question, I'm new to this stuff.

**Comment by Jabher** [comment]
Yup. That's an issue :D We were discussing it previously, and this is one of reasons to implement synaptic 2.

**Comment by vishiswoz** [comment]
Is there any temporary fix I can implement?

**Comment by Jabher** [comment]
I'm afraid I cannot give you any good solution right now, but as far as I remember it should not be that complicated, I've been trying to implement some dirty fix for that some time ago and it was quite simple.

**Comment by vishiswoz** [comment]
Is there any way I can create a new `Architect.Perceptron` from the `Network` I have and just copy over any key/data I need to?

**Comment by vishiswoz** [comment]
As an alternative, can I just convert the Architect.Perception using another JSON library and then convert it back?

**Comment by Peterard** [comment]
Bump
